browser/brave_shields/brave_shields_tab_helper.cc

```diff
@@ -260,8 +260,7 @@ CookieBlockMode BraveShieldsTabHelper::GetCookieBlockMode() {
     case ControlType::DEFAULT:
       break;
   }
-  NOTREACHED_IN_MIGRATION();
-  return CookieBlockMode::BLOCKED;
+  NOTREACHED();
```

Comment:
nit: `NOTREACHED() << "Unexpected value for control_type: "<< base::to_underlying(control_type);`

Reply:
Done